channels/iax2/parser.c

```diff
@@ -260,6 +260,15 @@ static void dump_versioned_codec(char *output, int maxlen, void *value, int len)
 	}
 }
 
+static void dump_call_token(char *output, int maxlen, void *value, int len)
+{
+	if (len == 0) {
+		strcpy(output, "Present");
```

Comment:
Why do we output "Present" when the length is 0? Just trying to make sense of how I should be interpreting this.

Reply:
Because the IE is present.

Reply:
> Because the IE is present.

To me, this makes it sound like the call token itself is present (and just not shown), not just the IE - it's clear that's present just from looking at the debug, or it wouldn't appear in the first place.

Personally, I think "Empty" would be a bit clearer to disambiguate "IE present, but no actual call token" from "IE present, with valid call token"... "Present" could imply the call token is there, just not dumped, unless someone were familiar with the output. That's my only suggestion.

Reply:
I agree that Empty would be clearer.